README.md

```diff
@@ -130,6 +130,14 @@ Note: This is different from the terraform reserved word `null` and is essential
 
 > see [example in aks-cft-deploy repo](https://github.com/hmcts/aks-cft-deploy/blob/main/azure-pipelines.yml)
 
+### components file location
+
+On repositories, it may not be possible to have components folder directy under the root of the repository. In such case, the `baseDirectory` option can be used so specify the location of the components folder.
+```yaml
+baseDirectory: "terraform/network/components"
+```
+> see [example in oracle-azure-infrastructure repo](https://github.com/hmcts/oracle-azure-infrastructure/blob/DTSPO-9224-pipeline-refactor/terraform/network/azure-pipelines.yml)
```

Comment:
🚫 do not link to non master references it will be deleted after merge, 

You could possibly just remove this link as external repository links are risky, someone might refactor this and not update this link, someone can easily search the HMCTS GitHub organisation to see how to use the parameter
```suggestion
> see [example in oracle-azure-infrastructure repo](https://github.com/hmcts/oracle-azure-infrastructure/blob/master/terraform/network/azure-pipelines.yml)
```

Reply:
Yupp - I was going to change it but wanted to make sure link works if you were going to press it :)